samples/ControlCatalog/MainView.xaml

```diff
@@ -24,14 +24,17 @@
       <TabItem Header="DataGrid"><pages:DataGridPage/></TabItem>
       <TabItem Header="DatePicker"><pages:DatePickerPage/></TabItem>
       <TabItem Header="Drag+Drop"><pages:DragAndDropPage/></TabItem>
+      <TabItem Header="Ellipse"><pages:EllipsePage/></TabItem>
```

Comment:
I'm pretty sure we have those on the Canvas page. Do we really need to bloat the left bar with extra pages?

Reply:
Helps a bit with testing, I was thinking about converting left bar to a tree view, so we can group similar content (like Geometry, Layout, Media, etc.). As currently it is not that readable either.

Reply:
I think the contents of these pages could be moved to the canvas page.